components/authz-service/README.md

```diff
@@ -615,6 +615,18 @@ For others, it will be necessary to add and remove policies
 to lock the system down so that known users are authorized to access
 specific resources.
 
+#### Adding New Default Policies
+
+To add a new default policy, the following is needed:
+
+1. A new IAM v1 policy: these are defined in [migrations](storage/postgres/migration/sql/),
+   and the [v1 constants package](constants/v1/constants.go#L3-L4).
+2. A new IAM v2 policy: they are defined in [server/v2/system](server/v2/system.go#L31).
```

Comment:
We should have them use the `datamigrations` for new V2 policies, otherwise they won't be applied to existing V2 installations.